aspnetcore/fundamentals/servers/aspnet-core-module.md

```diff
@@ -1,126 +1,59 @@
 ---
-title: ASP.NET Core Module
+title: ASP.NET Core Module for ASP.NET Core
 author: tdykstra
-description: Introduces ASP.NET Core Module (ANCM), an IIS module that lets the Kestrel web server use IIS or IIS Express as a reverse proxy server.
+description: Learn how the ASP.NET Core Module is an IIS module that allows the Kestrel web server use IIS or IIS Express as a reverse proxy server.
```

Comment:
?

Reply:
1. Latest guidance I have is that these descriptions aren't supposed to merely describe topic's content. They're to be written in active tone advertising to the reader what will happen if they read the topic. Therefore, I've been starting with phrases such as "Learn how to ...", "Discover how to ...", and "Find out how to ...".
2. Given that, the subject goes from *the topic* to *you*, and then the noun phrase becomes the object.
3. Kind'a (mostly) dropping "ANCM." See my remarks in https://github.com/aspnet/Docs/issues/5490#issuecomment-367886176.
4. Per the guidance I've been given, this doc set is to lean more formal (and less conversational) than the style manual dictates. Therefore, the word "lets" becomes "allows."

I'd be happy to revert this (or anything you like here). Just give the word.

Reply:
This sentence doesn't parse
>Learn how the ASP.NET Core Module is an IIS module that allows the Kestrel web server use IIS or IIS Express as a reverse proxy server.

"Learn about the ASP.NET Core Module, an IIS module that allows the Kestrel web server to use IIS or IIS Express as a reverse proxy server."

Reply:
Oh ... yes ... "TO" missing a "to" in there ...

> Learn how the ASP.NET Core Module is an IIS module that allows the Kestrel web server to use IIS or IIS Express as a reverse proxy server.

Reply:
I think my version is slightly more definitive and active.

Learn about something, where something is something that does something.

vs.

Learn how something is something that does something.

Shall we turn to @scottaddie to break the tie??!! :smile: lol

> Learn about the ASP.NET Core Module, an IIS module that allows the Kestrel web server to use IIS or IIS Express as a reverse proxy server.

or

> Learn how the ASP.NET Core Module is an IIS module that allows the Kestrel web server to use IIS or IIS Express as a reverse proxy server.